release.sh

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env bash
```

Comment:
both probably works, is `sh` to be preferred? The reason I use it for the other script is that alpine doesn't come with `bash`, just `sh`. But developer machines have both